images/image-repo-list-ws2019

```diff
@@ -0,0 +1,8 @@
+dockerLibraryRegistry: claudiubelu
```

Comment:
Is there a testgrid config somewhere that needs to reference this file now?

Reply:
If there's any, they can switch to using these files after the changes has been merged.